kube/resources/scheduler.yaml

```diff
@@ -204,20 +204,6 @@ spec:
             - name: gcs-log-creds-volume
               mountPath: /secrets/gcs-log-creds
               readOnly: true
-      initContainers:
-        - name: airbyte-seed
-          image: airbyte/seed
-          command: ["/bin/sh"]
-          args:
-            [
-              "-c",
-              "mkdir -p /configs/config && yes n | cp -r -i /app/seed/config /configs",
-            ]
-          volumeMounts:
-            - name: airbyte-volume-configs
-              mountPath: /configs
-            - name: airbyte-volume-workspace
-              mountPath: /workspace
```

Comment:
Is the `workspace` volume also related to the seeds?

Reply:
I don't think the workspace volume is related. This was probably put here unintentionally. Should be not problem removing since it's not used.